lib/mailer.js

```diff
@@ -10,6 +10,7 @@ const log = new Logger();
 const transporter = nodemailer.createTransport({
     host: process.env.MAIL_SERVER_HOST,
     port: process.env.MAIL_SERVER_PORT,
+    secure: process.env.MAIL_SERVER_TLS,
```

Comment:
Ooo - does this need to be set to anything in `.env`? My guess is that by default it will be `''` or `undefined`?

Reply:
Yep; it'll be `''` by default, the same as it currently is. The only difference is that if this is set to `yes` in `.env`, it can connect to the mailserver via TLS/SSL from the beginning, without STARTTLS. I hope I explained this clearly enough in the README.